hack/demo/prepare-alibaba-credentials.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+echo "accessKeyId: ${ALICLOUD_ACCESS_KEY}\naccessKeySecret: ${ALICLOUD_SECRET_KEY}" > alibaba-credentials.conf
```

Comment:
nitpick(security): Depending on the default umask, credentials will be temporarily readable by others. Not because I think this is super critical, but still because it's a potential security issue I wanted to mention it...

Reply:
Yes, so I deleted the file in the next line.